aspnet/dnx/commands.rst

```diff
@@ -1,8 +1,232 @@
+
+.. note:: The assembly listed in the ``commands`` section should be pulled in by a package that your application depends on.
+
+Running commands using dnx.exe
+------------------------------
+
+You can use DNX to run the commands defined by your project by entering the following in the command prompt from your project's directory::
+
+	dnx [command]
+
+You can also run commands from VS Code or Visual Studio. From VS Code, open the **Command Palette** (Ctrl+Shift+P) and enter the name of the command you want to run. From Visual Studio, open the **Command Window** (Ctrl+Alt+A) and enter the name of the command you want to run.
+	
+For example, the following command is used to run a web application::
+
+    dnx kestrel
+
+To run a console app, you can use the following command::
+
+	dnx run
+	
+.. note:: Note that running a command is short-hand for specifying the command assembly and it's arguments directly to DNX. For example, ``dnx web`` is a short-hand alias for ``dnx Microsoft.AspNet.Hosting hosting.ini``, where the hosting.ini file contains the command parameters. 
+
+Global commands
+---------------
+Global commands are DNX console applications (in a NuGet package) that are installed globally and runnable from your command line. Global commands are different from the commands that your add in the ``commands`` section of the *project.json* file of a project because global commands are not project specific. You can install, run, uninstall, build, and publish global commands. 
+
+Global commands use the local XML *NuGet.Config* file to store package source locations. The main sections for this file are ``packageRestore``, ``packageSources``, ``disabledPackageSources``, and ``activePackageSource``.
+
+Installing global commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can add a global command package and its dependencies using the `Package Manager Console <http://docs.nuget.org/consume/package-manager-console>`_. For example, to install the `SecretManager <http://www.nuget.org/packages/Microsoft.Framework.SecretManager>`_ from the **Package Manager Console**, enter the following::
```

Comment:
This is not correct. Using the package manager you just get the package for your project. No command will be installed. You can call directly `dnu commands install ...` and that will download the package and install it.